.build/customizations/sigip/init.sh

```diff
@@ -73,4 +73,13 @@ if [[ "$DROPSCHEMA" -eq 1 ]]; then
          -c "DROP SCHEMA IF EXISTS qwat_sigip CASCADE"
 fi
 
-PGSERVICE=${PGSERVICE} SRID=${SRID} ${DIR}/inserts.sh
+# create the qwat_sigip schema
+psql service=${PGSERVICE} -c "CREATE SCHEMA IF NOT EXISTS qwat_sigip;"
+
+# add the qwat_od.damage table
+psql service=${PGSERVICE} -v ON_ERROR_STOP=1 -v SRID=$SRID -f ${DIR}/damage/damage.sql
```

Comment:
Maybe for the example (this directory is used for work but also as an example/demonstration of customizations) I will put the addition of tables in a script like 'insert_tables.sh'.
Here it doesn't make much sense because there is only one table, but it's just for example.
I'll let you judge for yourself.

Reply:
Yes, I agree. But for now I just wanted to structure sigip in the same way as ch_vd_sire.